### PR TITLE
integrate compression middleware

### PR DESCRIPTION
**Pull request by hbakhtiyor**
need to find easy way to integrate middlewares, not just like this

### CODE REVIEW

File: src/ExpressModuleConfig.ts [hunk 1]
```diff
@@ -26,6 +26,59 @@ export interface ExpressModuleConfig {
...
+     * Options for compression middleware.
+     *
+     * See https://github.com/expressjs/compression for more information.
+     */
+    compression?: {
+
+        /**
+         * If set to true, compression middleware will be enabled.
+         */
+        enabled?: boolean;
```
Comment (pleerock) [inline src/ExpressModuleConfig.ts:38]: maybe this should be required option? Because otherwise neither of options are useful

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   ],
   "dependencies": {
     "body-parser": "~1.15.0",
+    "compression": "^1.6.1",
```
Comment (pleerock) [inline package.json:25]: I think need to find a minimal acceptable version for this to work. is 1.6.1 latest or minimal version?

File: src/ExpressModuleConfig.ts (hunk 1 above)
Comment (hbakhtiyor) [inline src/ExpressModuleConfig.ts:38]: right

> otherwise neither of options are useful

:confused:

File: package.json (hunk 2 above)
Comment (hbakhtiyor) [inline package.json:25]: it's latest version

what do you mean by minimal version? :confused:

**Pull request merged**